bpf/flows.c

```diff
@@ -224,18 +247,21 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
             bpf_ringbuf_submit(record, 0);
         }
     }
-
+    // finally, when flow receives FIN or RST, clean flow_directions
```

Comment:
Should we move this up a bit? There's an early return if the ringbuffer reserve fails, line 243? Maybe best is to put a goto from that early return so we don't have to worry about it.

Reply:
We can't because line 243 is in the `else` area. Cleanup will be omitted when the program finds flow in `aggregated_flows` map. 

But `goto cleanup` is the best idea. I will do it.

Reply:
done